advBatchControl.go

```diff
@@ -18,9 +18,7 @@ type ADVBatchControl struct {
 	ID string `json:"id"`
 	// RecordType defines the type of record in the block.
 	recordType string
-	// ServiceClassCode ACH Mixed Debits and Credits ‘200’
-	// ACH Credits Only ‘220’
-	// ACH Debits Only ‘225'
+	// Constant AutomatedAccountingAdvices: 280
```

Comment:
Should this be commented out?

Reply:
I'll remove!

Reply:
since this in adv control, I'll have it be this

	// This should be the same as BatchHeader ServiceClassCode for ADV: AutomatedAccountingAdvices.